algorithms/matrix/transpose/README.md

```
@@ -0,0 +1,156 @@
+# Transpose
+
+Take input text and output it transposed.
+
+Given an input text output it transposed.
+
+Roughly explained, the transpose of a matrix:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)